vcd/resource_vcd_vapp_emtpy_vm_test.go

```diff
@@ -182,7 +186,7 @@ func TestAccVcdVAppEmptyVm(t *testing.T) {
 					resource.TestCheckResourceAttrSet("vcd_vapp_vm."+netVmName1, "network.1.mac"),
 					resource.TestCheckResourceAttr("vcd_vapp_vm."+netVmName1, "network.1.connected", "true"),
 					// Ensuring adapter type stays intact after update
-					resource.TestCheckResourceAttr("vcd_vapp_vm."+netVmName1, "network.1.adapter_type", "E1000"),
+					resource.TestCheckResourceAttr("vcd_vapp_vm."+netVmName1, "network.1.adapter_type", "VMXNET3"),
```

Comment:
Does E1000 prevent having hot plug?

Reply:
this change needed due change of os_type and this one was needed cause some of them doesn't support hardware_version and hot add fields.